docusaurus/docs-internals/editor-controls/data-controls/imageUpload.md

```diff
@@ -0,0 +1,210 @@
+| `position?`                       | `number`            |       -       | The position of the control in toolbar                                                                                                                                                                                                                                                                                                                                                             |
+| `devices?`                        | `"all"` \| `"desktop"` \| `"responsive"`  | `"all"` | Define the devices where the control will be rendered. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices. |
+| `disabled?`                       | `boolean`           |    `false`    | Configure the condition under which the control is disabled or enabled.                                                                                                                                                                                                                                                                                                                            |
+| `config?.pointer`                 | `boolean`           |    `true`     | The pointer you see after adding an image serves as a visual indicator for determining the center of the image. By moving this pointer, you can accurately select where you want the center of the image to be positioned. This feature allows you to precisely align and position the image within the frame or designated area, ensuring it is perfectly centered according to your preferences. |
+| `config?.edit`                    | `boolean`           |    `true`     | Deactivate the possibility to delete an uploaded photo.                                                                                                                                                                                                                                                                                                                                                  |
+| `config?.disableSizes`            | `boolean `          |    `false`    | Allow you to add a dropdown with `Custom` and `Original` values that permit you to upload either the original image size or customize the image.                                                                                                                                                                                                                                                   |
+| `config?.acceptedExtensions`      | `Array<Extensions>` |       -       | Specifies the accepted types of extensions: <br/><br/> <b> Extensions = `"svg"` \| `"png"` \| `"jpg"` \| `"jpeg"` \| `"gif"` \| `"webp"` </b> |
```

Comment:
Specifies the accepted types of extensions: -> Specifies the accepted image extensions.